apache/flink
# [FLINK-8864] added command history

## What is the purpose of the change

*This PR adds history for sql queries/sqlclient commands*


## Brief change log

  - *Added history file for commands*

## Verifying this change
  - *Manually verification*

1. run sql-sclient, perform whatever queries, close close client
2. run sql-client again, check history (up, down, forward and backward searches via ctrl+s, ctrl+r)


## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)
